tools/compress.py

```diff
@@ -149,7 +162,10 @@ def compress_rom(
     compressed_rom_size = sum(
         align(len(segment.data)) for segment in compressed_rom_segments
     )
-    pad_to_multiple_of = 8 * 2**20  # 8 MiB
+    if pad_rom:
+        pad_to_multiple_of = 8 * 2**20  # 8 MiB
+    else:
+        pad_to_multiple_of = 4 * 2**10  # 4 KiB
```

Comment:
This should be 16 KiB to match the iQue NAND block size

Reply:
I think it would be nice to specify the multiple as the arg, but otherwise ique/n64 sounds good

Reply:
it's not just the multiple though, it's also padding with 0s vs padding with 0x00 0x01 0x02 etc

Reply:
Alright the multiple is now specified as `--pad-to` and the 0x00 0x01 pattern is called `--fill-padding-bytes`. Not a huge fan of the latter but I can't think of a better name